src/components/brands/BrandTile.jsx

```
@@ -0,0 +1,64 @@
+        )}
+        <div className="flex-1 min-w-0">
+          <div className="font-medium text-gray-900 truncate" title={name}>{name}</div>
+          <div className="text-xs text-gray-500 mt-1">{lastUpdateLabel}</div>
+        </div>
+        <div className="ml-2">
+          <button
+            type="button"
+            aria-label={`${isFollowing ? 'Unfollow' : 'Follow'} ${name}`}
+            onClick={(e) => {
+              e.stopPropagation();
+              try { track('brand_follow_toggle', { brandId: brand.id, action: isFollowing ? 'unfollow' : 'follow', via: 'brand_tile' }); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)